provider/internal/connector/connectoronboarding/resource.go

```diff
@@ -36,7 +36,7 @@ func (r *Resource) Metadata(ctx context.Context, req resource.MetadataRequest, r
 
 func (r *Resource) Schema(ctx context.Context, req resource.SchemaRequest, resp *resource.SchemaResponse) {
 	resp.Schema = schema.Schema{
-		MarkdownDescription: "Use this resource to wait for an SDC to finish onboarding. When an SDC is onboarded, either manually or using the CDO Terraform Modules for [AWS](https://github.com/CiscoDevNet/terraform-aws-cdo-sdc) and [vSphere](https://github.com/CiscoDevNet/terraform-vsphere-cdo-sdc), it can take a few minutes before the SDC is active and capable of proxying communications between CDO and the device. This resource allows you to wait until this is done.",
+		MarkdownDescription: "Use this resource to wait for an SDC to finish onboarding. When an SDC is onboarded, either manually or using the CDO Terraform Modules for [AWS](https://github.com/CiscoDevNet/terraform-aws-cdo-sdc) and [vSphere](https://github.com/CiscoDevNet/terraform-vsphere-cdo-sdc), it can take a few minutes before the SDC is active and capable of proxying communications between CDO and the device. This resource allows you to wait until this is done. If you see errors regarding public key not found, it is probably because you did not wait for this resource to finish before start utilizing the sdc_device, to fix this, add a `depends_on` block to your resource: `depends_on = [cdo_sdc_onboarding.example]`.",
```

Comment:
Please reword to: 
<br>*Note*: If you receive errors that read "public key not found", it is likely because you attempted to utilize the `cdo_sdc` resource before it has finished configuring itself, which is an asynchronous processor. To fix this, use the `cdo_sdc_onboarding` resource before utilizing the `cdo_sdc` to onboard any devices.

Reply:
done 👍